blog/_posts/2018-04-10-migrating-the-kubernetes-blog.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: blog
+title: 'Migrating the Kubernetes blog'
+date: '2018-04-10T22:20:00.000-07:00'
+author: zcorleissen
+---
+
+We recently migrated the Kubernetes blog from the Blogger platform to GitHub. With the change in platform comes a change in URL: formerly at [http://blog.kubernetes.io](http://blog.kubernetes.io), the blog now resides at [https://kubernetes.io/blog](https://kubernetes.io/blog).
+
+All existing posts redirect from their former URLs with `<rel=canonical>` tags, preserving SEO values.
+
+### How and why we migrated the blog
+
+Our primary reason for migrating was to make blog reviews easier, more streamlined, and more transparent. GitHub's pull request process makes it easier for multiple parties to collaborate on reviews. Blogger's web interface makes it difficult to open drafts to multiple reviewers without also granting access permissions and compromising security.
```

Comment:
We hope that by making the blog more easily accessible to the community, this will help generate more participation in both the review process and content contribution.

Reply:
@kbarnard10 Do you want this sentence added to what's there, or do you want it to replace some content?

Reply:
I think added unless it seems too repetitive to you

Reply:
@kbarnard10 I think the whole section needs to be revised; I'll work it in. 👍